javascript/react_js/advanced_concepts.md

```diff
@@ -19,7 +19,7 @@ By the end of this lesson, you should be able to do the following:
 * Explain Styled Components.
 * Explain Redux and state management systems.
 * Explain higher-order Components.
-* Explain Routers in web pages.
+* Explain how Context API works.
 * Explain how to make your own React Hooks.
```

Comment:
Can we have the Context API LO before the higher order components one, so that it's aligned with the order of topics in the lesson?

Reply:
alright!